test/e2e/e2e_test.go

```diff
@@ -72,4 +72,38 @@ var _ = Describe("LWS Operator", Ordered, func() {
 			return nil
 		}, 5*time.Minute, 5*time.Second).Should(Succeed(), "available condition is not found")
 	})
+	It("Verifying operator pod deleted and recovery", func() {
```

Comment:
If we delete the operator manually, I don't think that nobody recreates it again?. Instead maybe we should delete the operand pod and expect that operator recreates it?.

Reply:
Updated.